rts/Game/Camera/SpringController.cpp

```diff
@@ -164,13 +164,13 @@ float CSpringController::ZoomIn(const float3& curCamPos, const float2& zoomParam
 	return 0.25f;
 }
 
-float CSpringController::ZoomOut(const float3& curCamPos, const float2& zoomParams)
+float CSpringController::ZoomOut(const float3& curCamPos, const float3& newDir, const float& curDistPre, const float& scaledMode)
```

Comment:
I'm pretty sure `newDir` should be irrelevant in this function. Zoom out for this camera always originates from center of screen.

However, I don't know how to adjust for this.